docs/styles/display.md

```diff
@@ -0,0 +1,48 @@
+# Display
+
+The `display` property defines if a Widget is displayed or not. The default value is `"block"` which will display the widget as normal. Setting the property to `"none"` will effectively make it invisible.
```

Comment:
Doesn't it also affect the layout of other widgets? The Widget isn't invisible - it's no longer there at all. Might be worth expanding on that now or in the future in order to differentiate this from `visiblility: hidden`.

Reply:
It won't affect the layout if display is none. Will revisit this when there are docs that describe layout.